source/_patterns/01-molecules/item-details/item-details.twig

```diff
@@ -0,0 +1,39 @@
+{%- with list only -%}
+  {%- block list -%}
+
+    {%- set attributes = attributes|default({})|merge({class: attributes.class|default([])|merge(['item-detail-tag']) }) -%}
```

Comment:
`item` in `item-details-tag` is confusing. I understand it refers to the higher level thing for which this pattern is providing the detail tags, but as this pattern provides those detail tags in a list of items, we end up with BEM element names like `item-details-tag__item` ([line 13](https://github.com/libero/pattern-library/pull/57/files#diff-c6512084992b5859e968e96e8e1ba74eR13)).

Would a pattern name of `tags` work? Then you'd have BEM class names like `tags__item` and `tags__item_link`.

Reply:
I think that having this named as tag will be confusing and been seen as related to tag-list.

Reply:
Fair point.

Having gone round the houses with renaming this pattern, the best we've come up with so far is `content-meta`: it relates to content (not just articles), and contains some of the meta data for the content. It's not a great name though, so if someone's got a better suggestion please chip in.